### DISCUSSION TITLE
Unlock

### DISCUSSION THREAD

**Discussion by SerHB24** [cedricp/ddt4all#1219] [Q&A] [answered]
@cedricp Good morning, how can I contact you to unlock operational functions and where is the lock in the application? Regards

**Comment by Furtif** [comment, marked as answer]
for edit data fields you need unlock stuff 
![image](https://github.com/user-attachments/assets/0ca1171c-559a-47d8-aa73-39eb2da35346)

**Reply by SerHB24** [reply]
And how do I contact you to send you the request code?

**Comment by SerHB24** [comment]
And how do I contact you to send the request code?

**Reply by Furtif** [reply]
You need a DataBase [one here](https://github.com/cedricp/ddt4all/discussions/845) and resquest's is if you change data of some strufs. 

### But Pay attention:
![image](https://github.com/user-attachments/assets/2266d580-0f54-40bc-8f6c-185fb000945a)



#### **DDT4ALL TEAM or anyone who participates in the program are not responsible for their mistakes. or by the hardware they use (often not compatible, pelase read [Readme supported adapters](https://github.com/cedricp/ddt4all?tab=readme-ov-file#supported-diagnostic-adapters-so-far))**

Most requests are already in DataBase in this style: 
``` True, False ```

I strongly advise you to take photos or logs of anything you modify, so that you can return it to its original configuration.

**Reply by Furtif** [reply]
You should not contact us, you are solely responsible for what you do.

**Comment by cedricp** [comment]
There is no lock on DDT4All, I made it on ECUtweaker (Android version) a long time ago, but it's removed now so you can use it normally.
